Automattic/pocket-casts-android
# Bookmarks - Add multiselect on podcast page

## Description

This adds bookmarks multi-select on the podcast page.

## Testing Instructions
1. Launch the app
2. Play an episode and add a few bookmarks
3. Go to the corresponding Podcast Details page
4. Tap on bookmarks tab
5. Long press on a row
6. ✅ Verify you enter multi selection mode and the action bar appears
7. Tap on a row, ✅ verify the item is now selected
8. ✅ Verify the action bar selection count updated
9. ✅ Verify the edit option is visible
10. Tap the row again to deselect it, ✅ Verify the edit option is hidden
11. Tap Select All from more options in the multi-select toolbar
12. ✅ Verify all the items are selected
13. Tap and hold on a row again
14. ✅ Verify you see the 'Select All Above' , 'Select All ' and 'Select All Below' options
15. Tap each option and ✅ verify the correct items are selected
13. Tap and hold on a row again
14.  ✅ Verify you see the 'Deselect All Above', "Deselect All" and 'Deselect All Below' options
15. Tap each option and ✅ verify the correct items are selected

#### Regression

Test that select/ deselect options for the episodes tab works as before.

## Screenshots or Screencast 

Light Theme | Rose Theme
----|----
![Screenshot_20230728_165305](https://github.com/Automattic/pocket-casts-android/assets/1405144/806e4587-6da4-41a1-86a9-2fd08bd1304a) | ![Screenshot_20230728_165746](https://github.com/Automattic/pocket-casts-android/assets/1405144/121001a4-33ab-4474-bccc-1537cc672bfb)


## Checklist
 
#### I have tested any UI changes...
